ci_framework/roles/switch_config/tasks/onyx_config.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: Collect switch facts
+  mellanox.onyx.onyx_facts:
+  register: onyx_facts
+  failed_when: false
+
+- name: Check if JSON API is supported in switch
+  ansible.builtin.set_fact:
+    json_api_support: |-
```

Comment:
Even easier:
```
json_api_support: >-
      {{
          onyx_facts['rc'] == 1 and 'command \"json-print\"' in onyx_facts['module_stderr']
      }}
```

Reply:
I believe doing it the way you are suggesting will flip the expected results from False to True and vise versa.

Reply:
Oh true, I didn't realized the logic is negated. As simple as put a `not` at the very beginning. The suggestion here was mainly about merging the jinja in a single placeholder.